## This is the detected entities in an array structure
## Related pkg: visor
##
## Author: Ferdian Pratama

Header header
int16 count
sensor_msgs/Image[] entity
int16[] pos_minx
int16[] pos_maxx
int16[] pos_miny
int16[] pos_maxy